conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps_find_module_and_config.py

```diff
@@ -113,8 +113,17 @@ def test_reuse_with_modules_and_config(client):
     client.run("build . -if=install")
 
 
-@pytest.mark.parametrize("find_mode", ["both", "config", "module"])
-def test_transitive_modules_found(find_mode):
+find_modes = [
+    ("both", "both", ""),
+    ("config", "config", ""),
+    ("module", "module", ""),
+    ("both", None, ""),
+    ("both", None, "MODULE")
```

Comment:
This is the case that fails, that is fixed by the PR.
When:

* Consumer has `find_package(MYPKGB)` and `MODULES` are searched first by default (this either by specifying it in the `find_package` call, or by setting `CMAKE_FIND_PACKAGE_PREFER_CONFIG` to `OFF` (otherwise the `CMakeToolchain` generator sets it to `ON`).

* `MYPKGA` depends on `MYPKGB`
* `MYPKGB` only provides a `config` because it does not specify `cmake_find_mode` _at all_ (when it does, it all works fine).

Reply:
Do you mean ``MYPKGB`` depends on ``MYPKGA``? Seems typo, reversed.

Reply:
Oops, you're right. I've updated the comment !